addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -148,9 +148,8 @@
         <td class="o_wforum_view_count d-none d-lg-table-cell  text-center">
             <span t-out="question.views" t-attf-class="#{question.views == 0 and 'opacity-50'}" />
         </td>
-        <td class="o_wforum_last_activity d-none d-lg-table-cell text-center">
-            <!-- [TODO] write_date gives last date of ANY update including favouriting. Need a "last answer" date field. -->
-            <span t-field="question.write_date" t-options='{"format": "d MMM yy "}'/>
+        <td class="o_wforum_last_activity d-none d-lg-table-cell text-center" t-att-data-create-date="question.bump_date">
```

Comment:
Maybe `last-activity-date` (matching the css class)? So it can be assigned to `create` or `bump` depending on the case, without :question: -reading "create_date=bump_date"?

Reply:
Turning `t-att-data-create-date` to `t-att-data-last-activity`